app/src/androidTest/java/com/droidhats/campuscompass/views/MapFragmentTest.kt

```diff
@@ -54,12 +64,211 @@ class MapFragmentTest {
         }
     }
 
-    //Checking if the search bar exists
+
     @Test
     fun test_SearchBar() {
+        //Checks if the search bar with text "search" exists
+        val searchTextView = onView(
+            allOf(
+                withId(R.id.mt_placeholder), withText("Search"), isDisplayed()
+            )
+        )
+        searchTextView.check(matches(withText("Search")))
```

Comment:
I will use this as an example, but you will need to apply the same concept for every other val values that you have declared. 

In this case you assigned that whole onView to a variable named searchTextView then you use that variable to check if it matches the text "Search". After that, you no longer use that declared variable. There is no point in declaring variable that are not being used again. Instead, do the following:

```
        onView(
            allOf(
                withId(R.id.mt_placeholder), withText("Search"), isDisplayed()
            )
        ).check(matches(withText("Search")))
```
In the above example, the `val searchTextView =` has been removed and `searchTextView.check(matches(withText("Search")))` is now replaced with `).check(matches(withText("Search")))` where the `)` is the ending parentheses of the onView method.